src/components/form/row.vue

```diff
@@ -103,87 +130,66 @@ export default {
 .form-row {
   .table tbody & td { vertical-align: middle; }
 
-  // column widths
-  .name {
-    width: 500px;
-    max-width: 500px;
-    @include text-overflow-ellipsis;
+  td {
+    font-size: 16px;
+    padding: 3px 0px 3px 6px;
+    color: #333;
   }
 
-  .id-and-version {
-    width: 180px;
-    max-width: 180px;
+  .name {
+    font-size: 18px;
   }
 
-  .submissions {
-    width: 180px;
-    max-width: 180px;
+  .review-state, .total-submissions, .not-published {
+    a { @include text-link; }
+    text-align: right;
+    width: 100px;
+    & [class*='icon'] {
+      margin-left: 5px;
+      color: #888;
+    }
   }
 
   .actions {
-    width: 180px;
-    max-width: 180px;
+    text-align: center;
+    width: 100px,
   }
 
-  .name {
-    .link-if-can { font-size: 24px; }
+  .last-submission{
     a { @include text-link; }
-
-    .icon-angle-right {
-      font-size: 20px;
-      margin-left: 9px;
-    }
-
-    .form-icon {
-      font-size: 20px;
-      margin-right: 9px;
-      cursor: help;
-    }
-
-    .form-name-closed {
-      color: #999;
-    }
-
-    .form-icon-unpublished {
-      color: #999;
-    }
-
-    .form-icon-closed{
-      color: $color-danger;
-    }
-
-    .form-icon-closing{
-      color: $color-warning;
+    text-align: right;
+    width: 150px;
+    & [class*='icon'] {
+      margin-left: 5px;
+      color: #888;
     }
   }
 
-  .form-id, .version {
-    @include text-overflow-ellipsis;
-    font-family: $font-family-monospace;
+  .closing-icon {
+    margin-right: 5px;
+    color: #888;
   }
 
-  .version { color: #888; }
-
-  .submissions {
-    a { @include text-link; }
-    .icon-angle-right { margin-left: 6px; }
+  td.review-state {
+    background-color: #eee;
   }
 }
 </style>
 
 <i18n lang="json5">
 {
   "en": {
+    // This appears on a button linking to the draft testing form page
+    "test": "Test",
+    "unpublished": "Not published yet",
     // This text shows when the last Submission was received. {dateTime} shows
     // the date and time, for example: "2020/01/01 01:23". It may show a
     // formatted date like "2020/01/01", or it may use a word like "today",
     // "yesterday", or "Sunday".
     "action": {
       "fill": "Fill Form"
     },
-    "formClosedTip": "This Form is Closed. It is not downloadable and does not accept Submissions.",
-    "formClosingTip": "This Form is Closing. It is not downloadable but still accepts Submissions.",
-    "formUnpublishedTip": "This Form does not yet have a published version.",
+    "formClosingTip": "This Form is Closing. It is not downloadable but still accepts Submissions."
```

Comment:
Release criteria: "This Form is Closing and accepting its final Submissions. It is not downloadable but still accepts Submissions."??